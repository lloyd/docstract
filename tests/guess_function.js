/** if @@return appears, we should be able to guess
 *  it's a function */

/**
 * a guessable function documentation block
 * @returns {string} */
exports.getValue = someFuncRef;
