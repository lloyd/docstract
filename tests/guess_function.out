{
  "desc": "if @return appears, we should be able to guess\n it's a function", 
  "filename": "tests/guess_function.js", 
  "functions": [
    {
      "desc": "a guessable function documentation block", 
      "name": "getValue", 
      "returns": {
        "type": "string"
      }
    }
  ], 
  "module": "guess_function"
}
